src/System.Runtime.Serialization.Primitives/tests/System/Runtime/Serialization/CollectionDataContractAttributeTests.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.Serialization.Tests
+{
+    public class CollectionDataContractAttributeTests
+    {
+        [Fact]
+        public void Ctor_Default()
+        {
+            var attribute = new CollectionDataContractAttribute();
+            Assert.False(attribute.IsReference);
+            Assert.False(attribute.IsReferenceSetExplicitly);
+            Assert.Null(attribute.ItemName);
+            Assert.False(attribute.IsItemNameSetExplicitly);
+            Assert.Null(attribute.Name);
+            Assert.False(attribute.IsNameSetExplicitly);
+            Assert.Null(attribute.Namespace);
+            Assert.False(attribute.IsNamespaceSetExplicitly);
+            Assert.Null(attribute.KeyName);
+            Assert.False(attribute.IsKeyNameSetExplicitly);
+            Assert.Null(attribute.ValueName);
+            Assert.False(attribute.IsValueNameSetExplicitly);
+        }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void IsReference_Set_GetReturnsExpected(bool value)
+        {
+            var attribute = new CollectionDataContractAttribute() { IsReference = value };
+            Assert.Equal(value, attribute.IsReference);
+            Assert.True(attribute.IsReferenceSetExplicitly);
+        }
+
+        [Theory]
+        [InlineData(null)]
```

Comment:
can combine all the InlineData for all the tests into separate ```theoryData<string>``` property

Reply:
Fixed